pom.xml

```diff
@@ -478,6 +478,13 @@
             <version>4.5.5-3.0</version>
         </dependency>
 
+        <!-- p4 scm plugin -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>p4</artifactId>
```

Comment:
I don't actually see any new java code, so is this actually a requirement?

Reply:
You are right. Took that one out.